modules/microshift-otel-verify.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift_telemetry-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-otel-verify_{context}"]
+= Enabling {microshift-short} Observability
+
+After updating the configuration file and installing the service, enable the {microshift-short} observability service by using one of the following methods:
+
+* Restarting the {microshift-short} service.
+* Rebooting the host machine.
+
+.Restart the {microshift-short} service:
+
+[source,terminal]
+----
+$ sudo systemctl restart microshift-observability
+----
+
+.Reboot the host machine:
+
+[source,terminal]
+----
+$ sudo systemctl reboot
+----
```

Comment:
I'm not quite sure if we need to recommend that.
It's true for MicroShift because of many moving parts, but for otel we should be good with just restarting the service